docs/userGuide/syntax/breadcrumbs.md

```diff
@@ -0,0 +1,15 @@
+
+
+## Breadcrumbs
+
+Breadcrumb components provide an easy way for readers to navigate the hierarchy of the site.
+
+#### Usage
+
+Simply include the breadcrumb component (`<breadcrumb />`) into the page you want and breadcrumbs for that page will be automatically generated.
+
+You can also insert the breadcrumb component into a layout file to generate breadcrumbs for all pages using that layout.
+
+<box type="tip">
+  Breadcrumb components refer to the Site Navigation for the hierarchy of pages. <strong>Breadcrumbs will not appear if there is no Site Navigation present on the page!</strong>
+</box>
```

Comment:
Would it be useful to show an example of what breadcrumbs look like (perhaps as shown in the original issue #2094)?

Reply:
Hi @lhw-1! I tried doing that using the `codeAndOutputSeparate` boilerplate but it doesn't show because i think the breadcrumb does not work with includes. I will add an example without using the boilerplate!